### ISSUE TITLE
It should be possible to enable explicit ssl mode for file transfers with FTPS

### ISSUE THREAD

**Issue by anzecesar** [akka/alpakka#976]
Listing files on a FTPS server that requires encryption for data files fails without setting `ftpClient.execPROT("P")`.

The server returns
```
LIST /
431 The data connection must be SSL-secured. Please configure your client to use explicit or implicit SSL for data connections.
```

of secondary concern is also, that this error stays silent and the stream terminates with success, with 0 files listed.

I would like to add it to [FtpOperations trait](https://github.com/akka/alpakka/blob/a24bbb40f246ab61cd5be34527429c8991de88f1/ftp/src/main/scala/akka/stream/alpakka/ftp/impl/FtpOperations.scala#L19-L32), but that requires possibly breaking changes to the method signature. 

FtpSettings are then not really common between FTP and FTPS, if `FtpsSettings` gain e.g. `explicitSSL`.

I'm happy to do it, if someone can point me in the right direction.

**Comment by anzecesar** [comment]
I've implemented this here: https://github.com/akka/alpakka/pull/977

**Comment by ennru** [comment]
Fixed with #977
